packages/client/src/v2-events/features/events/useCertificateTemplateSelectorFieldConfig.ts

```diff
@@ -31,6 +37,12 @@ export const useCertificateTemplateSelectorFieldConfig = (
     )?.id,
     options: certificateTemplates
       .filter((x) => x.event === eventType)
+      .filter((x) => {
+        if (x.conditionals) {
+          return areConditionsMet(x.conditionals, declation)
+        }
+        return true
+      })
```

Comment:
```suggestion
      .filter((x) => !x.conditionals || areConditionsMet(x.conditionals, declation))
```
You could simplify this